README.md

```diff
@@ -328,7 +329,7 @@ The **pointfinder.tsv** output file generated by `staramr` contains the followin
 The **plasmidfinder.tsv** output file generated by `staramr` contains the following columns:
 
 * __Isolate ID__: The id of the isolate/genome file(s) passed to `staramr`.
-* __Gene__: The particular plasmid gene detected.
+* __Plasmid__: The particular plasmid gene detected.
```

Comment:
Can you change **The particular plasmid *gene* detected.** => **The particular plasmid *type* detected.**.

Also, for the `pointfinder.tsv` section of this document (above) there's some extra **()**:

> * Gene: The particular AMR gene detected, with the point mutation within **()**.

Reply:
Removed in 22b711b